### DISCUSSION TITLE
ChakraUI, Panda and Ark UI

### DISCUSSION THREAD

**Discussion by xeinebiu** [chakra-ui/chakra-ui#8465] [General]
Hey Chakra fam,

Hope all's good!

So, at our company, we're all about ChakraUI for our components. I'm curious about where Chakra's headed. Heard about Panda and ArkUI, but not sure how they fit in.

1. Is Chakra v3 happening?
2. Panda in Chakra's future?
3. Is ArkUI the new ChakraUI? 

Let's chat!

**Comment by veloware** [comment]
https://www.adebayosegun.com/blog/the-future-of-chakra-ui

https://www.adebayosegun.com/blog/chakra-panda-ark-whats-the-plan
